### PR TITLE
Add manage page and delete fuction for blogposts

### PR DESCRIPTION
**Pull request by TheMarv**

### CODE REVIEW

File: pages/blog/index.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import TableHead from '@mui/material/TableHead';
+import EditIcon from '@mui/icons-material/Edit';
+import IconButton from '@mui/material/IconButton';
+import DeleteIcon from '@mui/icons-material/Delete';
+import DialogTitle from '@mui/material/DialogTitle';
+import useHydration from '../../hooks/useHydration';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import TableContainer from '@mui/material/TableContainer';
+import Link from 'next/link';
```
Comment (marcbendixen) [inline pages/blog/index.jsx:18]: Just a hint: you could use "Organize imports" from VSCode (via command palette) to automatically sort them for a better overview.

File: pages/blog/index.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+              blogposts.map(blogpost => (
+                <TableRow key={blogpost.id}>
+                  <TableCell>{blogpost.id}</TableCell>
+                  <TableCell>{blogpost.title}</TableCell>
+                  <TableCell sx={{ width: '112px' }} align="right">
+                    <IconButton
+                      color="secondary"
+                      variant="outlined"
+                      onClick={() => {
+                        addAlert('Currently not implemented', 'error');
```
Comment (marcbendixen) [inline pages/blog/index.jsx:52]: In general: if something isn't implemented yet, it should not be part of an PR.

**Review by marcbendixen — APPROVED** [review]
LGTM 🚀

File: pages/blog/index.jsx (hunk 1 above)
Comment (TheMarv) [reply pages/blog/index.jsx:18]: I try to sort my imports by the length of the line

**Pull request merged**